### DISCUSSION TITLE
Legacy Unenrollment/Dev Mode exploits

### DISCUSSION THREAD

**Discussion by legendaryfishwastaken** [3kh0/ext-remover#1122] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Alright, so basically, I have this really old Chromebook from 2014 i got from e-waste management, and I’d like to use it as a regular laptop, because why not. It stopped recieving updates 5 years ago. The thing is, however, it’s an enrolled chromebook. I have no contact with the school board who used to own it, so I can’t tell them to unenroll it. All that I have been able to do so far is sign in, using an account and password that was for some reason taped on the front (by the way, even this was difficult. the wifi refused to cooperate for a while). If anyone has any idea how I can  unenroll, please let me know.

Model: Acer Chromebook 13 (CB5-311, C810)
Board: nyan_big

**Comment by AshtonDavies** [comment]
Alright, so this is obviously the one kind of situation where it is totally legal to unenroll a managed device from a school or workplace, which is good 👍😊.

For a newer model, I would suggest CryptoSmite or Pencil Method, but you can definitely use [Sh1mmer](https://sh1mmer.me) for this one if your board is hosted somewhere (you can check the [Sh1mmer Discord server](https://discord.com/invite/GtJwBrEd7p)), because it lost support 5 years ago, which means that it has not received the patch.

**Reply by legendaryfishwastaken** [reply]
There is no leaked RMA shim for nyan_big. I can't use sh1mmer.

**Reply by S-PScripts** [reply]
nyan_big was only used for one chromebook which is unfortunate. it is a code name and a board name

octopus, nissa, brya, coral and dedede (board names) are the most popular for chromebooks
https://chrome100.dev/

if you manage to find its leaked rma shim, you could use these
https://darkn.bio/sh1mmer/builder
https://sh1mmer.diffusehyperion.com/builder/builder.html

**Reply by s0urce-c0de** [reply]
Do you have a a screenshot of chrome://management on Chromebook?

**Reply by simpansoftware** [reply]
If crosh is unblocked you can probably use https://coolelectronics.me

**Reply by s0urce-c0de** [reply, marked as answer]
Try https://github.com/S-PScripts/chromebook-utilities/blob/main/Exploits/Unenrollment%20(Crosh)%20(to%20v100)

**Reply by Exploit-Master122** [reply]
I have the Dedede board
